### PR TITLE
Plurals for Consistency suggestions

### PR DESCRIPTION
**Pull request by vlad-timotei**
1. In order to use async/await we need to upgrade to es2017. Fixing various es2017 things.
2. In order to use `gd_user` object we need to load glotdict script earlier. 
3. Implements plurals for consistency and a message for GTEs/PTEs if there are multiple consistency alternatives.

Testing needed.
Fixes: #330 

# Demos:

## Singular string - contributor view
demo_singular_contributor

## Singular string - PTE/GTE view
demo_singular_PTE-GTE

## Plural string - contributor view
demo_plural_contributor

## Plural string - PTE/GTE view
demo_plural_PTE-GTE

## Plural string - a very long list
demo_plural_long_list

### CODE REVIEW

File: js/init.js
Comment (Mte90) [inline js/init.js:1]: maybe consistency is the case that is loaded before then glotdict general :-)

**Review by Mte90 — CHANGES_REQUESTED** [review]

File: js/init.js
Comment (vlad-timotei) [reply js/init.js:1]: In order to use `gd_user object` that is declared in glotdict.js we need to load glotdict.js script before.
That was the reason behind.

Comment (Mte90) [reply js/init.js:1]: Ok so I am wondering if it is the case to have a file with all those definitions to run as first and later the rest.

Comment (Mte90) [reply js/init.js:1]: yes it is the best way to achieve it :-)

**Comment by Mte90** [comment]
Waiting for a review by @webaxones :-)

**Pull request merged**
